pkg/operator/controller/machineconfig/constants.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package machineconfigcontroller
+
+import (
+	"time"
+)
+
+const (
+	// finalizer name for nodeobservabilitymachineconfig resources
+	finalizer = "NodeObservabilityMachineConfig"
+
+	// defaultRequeueTime is the default reconcile requeue time
+	defaultRequeueTime = 3 * time.Minute
+
+	// Null is defined for empty string
+	Null = ""
```

Comment:
that's confusing
it's either nil or "" in go

Reply:
changed from Null to Empty.